pkg/hubble/relay/queue/priority_queue.go

```diff
@@ -17,20 +17,20 @@ package queue
 import (
 	"container/heap"
 
-	"github.com/cilium/cilium/pkg/hubble/api/v1"
+	observerpb "github.com/cilium/cilium/api/v1/observer"
```

Comment:
Is hubble-relay requesting flows from the local node where it's running? If yes, the priority queue needs to understand the local events from the local ring buffer.

Reply:
I don't understand this. Hubble-relay requests flows from all nodes and I don't see a reason to differentiate the "local" one. Mind expanding on this?

Reply:
In the previous incarnation, relay and hubble was the same process (node could proxy peer traffic). In the current setup @aanm they are completely two different pods, so local flows still need to be fetched from another pod